README.md

```diff
@@ -0,0 +1,36 @@
+# Lost
+
+Are you truly lost? Will you ever return home?
+A researcher on a space expedition is stranded on an alien world with no way to get home.  
+
+ ## How To Play
+
+Download the Lost module and it's dependencies. When this is installed, create a new world with the Lost template. If everything worked, you should spawn in a world with a small house nearby.
+
+## Current Features
+
+### Special Structure Generation
+
+As you explore you might find buildings that contain treasures, secerts and puzzels.
```

Comment:
Another typo. It really helps to use an editor that automatically checks for spelling and even grammar / issues like the its vs it's bit - even if you're editing in a browser you can usually get addons for that :-)